manifests/params.pp

```diff
@@ -1,9 +1,10 @@
 # Set default parameters based on distribution release
 class nfs::params {
   $portmap = $::operatingsystem? {
-    'Debian' => $::lsbdistcodename? {
-      'Wheezy' => 'rpcbind',
-      default  => 'portmap',
+    'Debian' => versioncmp($::operatingsystemmajrelease, 7)? {
```

Comment:
> `versioncmp` returns a number, not a string.

Uh, indeed. Thanks, and sorry for blindly trusting Mickaël's advice.

> Use this instead:

> ```puppet
> 'Debian' => versioncmp($::operatingsystemmajrelease, 7) >= 0 ? {
>   true => 'rpcbind',
>   false => 'portmap',
> }
> ```

I tried this (modulo I quoted '7'), and I get:

```
err: Could not retrieve catalog from remote server: Error 400 on SERVER: Syntax error at '>='; expected '}' at /etc/puppet/environments/production/modules/nfs/manifests/params.pp:4 on node xxx
```